monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -48,7 +48,7 @@ link:#thanosrulerconfig[ThanosRulerConfig]
 
 [options="header"]
 |===
-| Property | Type | Description 
+| Property | Type | Description
 |apiVersion|string|Defines the API version of Alertmanager. Possible values are `v1` or `v2`. The default is `v2`.
```

Comment:
@bburt-rh, could this be the case?
```suggestion
|apiVersion|string|Defines the API version of Thanos Ruler. Possible values are `v1` or `v2`. The default is `v2`.
```
And the same on some of the following lines.